surveys/urls.py

```diff
@@ -10,4 +10,9 @@
         views_api.SurveyPostView.as_view(),
         name='survey-post',
     ),
+    url(
+        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>\w+)?$',
```

Comment:
Do we know if a `user_id` can contain other character than a-z, can it contain underscores or dashes?